packages/flutter_list_bloc/test/src/continuous_listview_bloc_test.dart

```diff
@@ -0,0 +1,110 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:flutter_list_bloc/flutter_list_bloc.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:list_bloc/list_bloc.dart';
+import 'package:mocktail/mocktail.dart';
+import '../utils/empty_test_widget.dart';
+import '../utils/test_app_container.dart';
+import '../utils/test_error_widget.dart';
+import '../utils/test_item.dart';
+import '../utils/test_loading_indicator.dart';
+
+class MockFilter extends Mock implements OffsetLimitFilter {}
+
+void main() {
+  group('ContinuousListViewBlocBuilder test', () {
+    const testDataText = 'TEST DATA';
+
+    final controller = ScrollController();
```

Comment:
@maksganzol Need to dispose this, right?

Reply:
Added dispose after tests.